src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  };
+};
+
+Animal.__proto__.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  };
+
+  hide() {
+    this.hidden = !this.hidden;
+  };
+};
 
 class Carnivore extends Animal {
-  // write your code here
-}
+  bite(herbivore) {
```

Comment:
```suggestion
  bite(prey) {
```

`herbivore instanceof Herbivore` looks strange